src/components/homePage/Hero.vue

```diff
@@ -74,9 +77,25 @@ export default {
 	}
 }
 
+button {
```

Comment:
Please also add a `button:focus` style to prevent the weird border that appears after it's clicked 

![image](https://user-images.githubusercontent.com/41529410/111908171-9782ea00-8a2e-11eb-8d4f-86129380814e.png)